### PR TITLE
Update proto submodule in prep for ShutdownWorker API call

### PR DESCRIPTION
**Pull request by yuandrew**
## What was changed

Ran `git pull --squash --rebase=false -s subtree ssh://git@github.com/temporalio/api.git master --allow-unrelated-histories` from the README to update the submodule and fix up the new rust errors from the update.

## Why?

Need to update the proto submodule to access ShutdownWorker API for #822

### CODE REVIEW

File: client/src/raw.rs [hunk 1]
```diff
@@ -1124,6 +1124,52 @@ proxier! {
             r.extensions_mut().insert(labels);
         }
     );
+    (
+        shutdown_worker,
+        ShutdownWorkerRequest,
+        ShutdownWorkerResponse,
+        // TODO: need the namespaced_request label?
+        |r| {
```
Comment (yuandrew) [inline client/src/raw.rs:1132]: @Sushisource I added these in by default to match the majority, but lmk if this should be removed

Comment (Sushisource) [reply client/src/raw.rs:1132]: Yeah it should be kept. All namespaced calls should have this label.

**Review by Sushisource — APPROVED** [review]
Sweet, thanks!

File: client/src/raw.rs (hunk 1 above)
Comment (yuandrew) [reply client/src/raw.rs:1132]: oops, forgot to remove this comment, will remove in the follow-up PR

**Pull request merged**